models/model.js

```diff
@@ -53,7 +58,9 @@ exports.getArticles = async (topic) => {
   try {
     if (topic.topic) {
```

Comment:
what if the topic was 'DROP TABLE'? You'd lose your data. You should probably reuse a previously created model in order to double check that your topic is a valid topic 👀